### PR TITLE
Make blue skin into an elf

### PR DESCRIPTION
**Pull request by LoneWolfHT**
ElfCTF

### CODE REVIEW

**Comment by LoneWolfHT** [comment]
Don't merge please, working on another skin...

**Comment by LoneWolfHT** [comment]
Ready for merge

**Comment by Thomas--S** [comment]
I'd prefer it if the red parts would be in another color, probably silver-ish. This will lead to less confusion with the team colors.

**Comment by LoneWolfHT** [comment]
image
How's this?

**Comment by Thomas--S** [comment]
I think this looks much better.

The only thing that "disturbes" me a little bit are the black things on the hands? Should these be finger-less gloves?

**Comment by LoneWolfHT** [comment]
Yeah. Those are pretty well known among the players I think. I personally like em

**Comment by LoneWolfHT** [comment]
Changed eye color
image

**Comment by GreenXenith** [comment]
I suggest updating the credits section in the README to reflect this change.
Something like ```- `ctf_colors_skin_blue.png` based on https://minecraft.novaskin.me/skin/3232713028, modified by LoneWolf```.

**Comment by LoneWolfHT** [comment]
@Thomas--S @rubenwardy ready for review/merge

File: README.md [hunk 1]
```diff
@@ -8,12 +8,13 @@ All content must be freely licensed.
...
 ### Code
 * `skins` MIT by rubenwardy
-* `snow` MIT by LoneWolf, rubenwardy, Thomas-S, and GreenXenith
-* `winterize` MIT by LoneWolf
+* `snow` MIT by LoneWolfHT, rubenwardy, Thomas-S, and GreenXenith
+* `winterize` MIT by LoneWolfHT
 
 ### Media
+
 * `Santa` CC BY-SA 3.0 by jordan4ibanez
```
Comment (ClobberXD) [inline README.md]: This line doesn't apply to both the skins in common now, and can be removed(?)

Comment (GreenXenith) [reply README.md]: Yes, this should be changed to ```* `Skins` ```.

Comment (LoneWolfHT) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ All content must be freely licensed.
 
 ### Media
 
-* `Santa` CC BY-SA 3.0 by jordan4ibanez
+* `Skins`
```
Comment (ClobberXD) [inline README.md:16]: `skins`, not `Skins` ;)

Or, just remove the backticks. Your choice.

Comment (GreenXenith) [reply README.md:16]: That would be inconsistent with the others.

Comment (GreenXenith) [reply README.md:16]: I think this PR should be merged as-is and a separate one be made to fix _all_ of the media credits.

Comment (ClobberXD) [reply README.md:16]: Hmm... it seems like all the names under `Media` are inconsistent. In CTF documentation, backticks are used for mod names or other pieces of actual code (see the `Code` section for an example).

Comment (ClobberXD) [reply README.md:16]: > I think this PR should be merged as-is and a separate one be made to fix all of the media credits.

Agreed

Comment (LoneWolfHT) [reply README.md:16]: Up to you 🤷‍♀️

**Review by ClobberXD — APPROVED** [review]
New skin looks much better :+1:

**Pull request merged**